modules/microshift-greenboot-check-status.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * microshift_running applications/checking-greenboot-scripts-status.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-greenboot-check-status_{context}"]
+= Checking the status of Greenboot health checks
+
+Check the status of Greenboot health checks before making changes to the system or during troubleshooting. You can use any of the following several commands to help you ensure that Greenboot scripts have finished running.
+
+.Procedure
+
+* To see a report of health check status, such as "active" if Greenboot is still running and "inactive" if it has exited, use the following command:
++
+[source,terminal]
+----
+$ systemctl is-active greenboot-healthcheck.service
+----
+
+* To see a report of health check status, such as "running" while the health checks are still running, or "exited" when the health checks are done, use the following command:
++
+[source,terminal]
+----
+$ systemctl show --property=SubState --value greenboot-healthcheck.service
+----
+
+* To see a report of a successful Greenboot check, such as "success" when the checks have passed, use the following command:
++
+[source,terminal]
+----
+$ systemctl show --property=Result --value greenboot-healthcheck.service
```

Comment:
This command prints out `exit-code`

` [redhat@dhcp-1-235-125 ~]$ systemctl show --property=Result --value greenboot-healthcheck.service
exit-code`

Reply:
~~`systemctl show --property=ExecMainStatus --value greenboot-healthcheck.service` will output `1` if greenboot failed, and `0` if succeeded.~~ I see it's included below.

Maybe we can just document `SubState`? I know it's not the most intuitive, but I don't see anything really intuitive
- start == running
- exited == success
- failed == failure

Reply:
@pmtk Would that look like:

`$ systemctl show --property=SubState--value greenboot-healthcheck.service`

Reply:
But we already have it on [line 24](https://github.com/openshift/openshift-docs/pull/65191/files#diff-4d7cd104ea0675a25fbf6df6cc70195e8864f64b6e7e9ab2ceac7f62e2d93b22R24) of this PR.
We'd just need to extend the explanation of start, exited, and failed